conduit/plugins/processors/filterprocessor/gen/generate.go

```diff
@@ -296,7 +296,7 @@ func main() {
 	}
 
 	// Process fields.
-	fields, errors := getFields(sdk.SignedTxnInBlock{}, ignoreTags)
+	fields, errors := getFields(sdk.SignedTxnWithAD{}, ignoreTags)
```

Comment:
Switched to `SignedTxnWithAD` because that is the type used for inner transactions. This way the same lookup function is used for the root transaction and the inner transactions.